Person/Person/main.m

```diff
@@ -51,13 +59,55 @@ - (NSString *)phoneNumber {
     return _phoneNumber;
 }
 
-@end
+- (BOOL)checkSameCity:(NSString *)city {
```

Comment:
This is 100% correct. You could make it a bit more concise by just return the result of the `isEqualToString` method call:

```objective-c
- (BOOL)checkSameCity:(NSString *)city {
  return [_city isEqualToString:city];
}
```